test/image/mocks/date_axes.json

```diff
@@ -2,16 +2,89 @@
   "data": [
     {
       "x": [
-        "2013-10-04 22:23:00",
-        "2013-11-04 22:23:00",
-        "2013-12-04 22:23:00"
+        "1900-01-01",
+        "2000-01-01",
+        "2100-01-01"
       ],
-      "y": [
-        1,
-        3,
-        6
+      "y": [1, 3, 2]
+    },
+    {
+      "x": [
+        "2013-05-01",
+        "2013-09-01",
+        "2014-01-01"
+      ],
+      "y": [1, 3, 2],
+      "xaxis": "x2",
+      "yaxis": "y2"
+    },
+    {
+      "x": [
+        "2013-11-17",
+        "2013-12-15",
+        "2014-01-12"
+      ],
+      "y": [1, 3, 2],
+      "xaxis": "x3",
+      "yaxis": "y3"
+    },
+    {
+      "x": [
+        "2013-01-01",
+        "2013-01-02",
+        "2013-01-03"
+      ],
+      "y": [1, 3, 2],
+      "xaxis": "x4",
+      "yaxis": "y4"
+    },
+    {
+      "x": [
+        "2013-07-01 18:00",
+        "2013-07-02 00:00",
+        "2013-07-02 06:00"
+      ],
+      "y": [1, 3, 2],
+      "xaxis": "x5",
+      "yaxis": "y5"
+    },
+    {
+      "x": [
+        "2013-01-01 23:59",
+        "2013-01-02 00:00",
+        "2013-01-02 00:01"
+      ],
+      "y": [1, 3, 2],
+      "xaxis": "x6",
+      "yaxis": "y6"
+    },
+    {
+      "x": [
+        "2013-07-01 23:59:59",
+        "2013-07-02 00:00:00",
+        "2013-07-02 00:00:01"
       ],
-      "type": "scatter"
+      "y": [1, 3, 2],
+      "xaxis": "x7",
+      "yaxis": "y7"
     }
-  ]
+  ],
+  "layout": {
+    "showlegend": false,
+    "width": 600,
+    "height": 500,
+    "yaxis": {"domain": [0, 0.04]},
+    "yaxis2": {"domain": [0.16, 0.2]},
+    "yaxis3": {"domain": [0.32, 0.36]},
+    "yaxis4": {"domain": [0.48, 0.52]},
+    "yaxis5": {"domain": [0.64, 0.68]},
+    "yaxis6": {"domain": [0.80, 0.84]},
+    "yaxis7": {"domain": [0.96, 1]},
+    "xaxis2": {"anchor": "y2"},
+    "xaxis3": {"anchor": "y3"},
+    "xaxis4": {"anchor": "y4"},
+    "xaxis5": {"anchor": "y5"},
+    "xaxis6": {"anchor": "y6"},
+    "xaxis7": {"anchor": "y7"}
```

Comment:
@etpinard good point about including the UTC behavior in the mocks. I replaced the simple `date_axes.json` with this one that includes tick scales all the way from full years to partial seconds.